IPython/testing/tools.py

```diff
@@ -10,6 +10,7 @@
 # Distributed under the terms of the Modified BSD License.
 
 import os
+from pathlib import Path
```

Comment:
There are plenty of places, where os can be replaced, I want to keep this PR short, therefore I ignore them.